util/ident/ident.go

```diff
@@ -47,7 +47,7 @@ func DefaultUserAgent() string {
 
 // DefaultPeerID return default PeerID
 func DefaultPeerID() string {
-	return "-GT0001-"
+	return "-LT1100-"
```

Comment:
This was Peer ID of anacrolix/torrent but we use libtorrent now so i changed it accordingly.